### PR TITLE
chore(docs): TS getting started doesn't compile

### PR DESCRIPTION
**Pull request by campionfellin**
Closes https://github.com/awslabs/cdk8s/issues/386

Tested locally, now if you copy + paste the example here it should all compile.

Signed-off-by: campionfellin <campionfellin@gmail.com>

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by anthonydahanne** [comment]
hello, I have created a similar PR for Java : #407.
Now I guess that you would also need to update the `WebService` part of the documentation as well (some Kube* API are also referenced)

File: docs/getting-started.md [hunk 1]
```diff
@@ -212,23 +212,23 @@ resources inspired by [paulbouwer](https://github.com/paulbouwer)'s
     import { App, Chart } from 'cdk8s';
 
     // imported constructs
-    import { KubeDeployment, KubeService, IntOrString } from './imports/k8s';
+    import { Deployment, Service, IntOrString } from './imports/k8s';
```
Comment (iliapolo) [inline docs/getting-started.md:215]: The latest version actually has the `Kube` prefix for all of these. Sure you tested with the latest import?

Comment (campionfellin) [reply docs/getting-started.md:215]: Great point, I was using `0.33.0`, seems to be fixed in latest version. Though also noticing now that latest version is not publishing to Brew... so I'll make a different issue for that

Comment (campionfellin) [reply docs/getting-started.md:215]:
```suggestion
    import { KubeDeployment, KubeService, IntOrString } from './imports/k8s';
```

File: docs/getting-started.md [hunk 2]
```diff
@@ -212,23 +212,23 @@ resources inspired by [paulbouwer](https://github.com/paulbouwer)'s
...
 
-    class MyChart extends Chart {
+    export class MyChart extends Chart {
       constructor(scope: Construct, name: string) {
         super(scope, name);
 
         const label = { app: 'hello-k8s' };
 
-        new KubeService(this, 'service', {
+        new Service(this, 'service', {
```
Comment (campionfellin) [inline docs/getting-started.md:223]:
```suggestion
        new KubeService(this, 'service', {
```

File: docs/getting-started.md [hunk 3]
```diff
@@ -228,7 +228,7 @@ resources inspired by [paulbouwer](https://github.com/paulbouwer)'s
           }
         });
 
-        new KubeDeployment(this, 'deployment', {
+        new Deployment(this, 'deployment', {
```
Comment (campionfellin) [inline docs/getting-started.md:231]:
```suggestion
        new KubeDeployment(this, 'deployment', {
```

**Review by iliapolo — APPROVED** [review]

**Pull request merged**